test/integration/test_load_and_run_checkpoint.py

```diff
@@ -57,11 +57,13 @@
 
 _DEPRECATED_MODEL_INFO = [
     # model card: https://huggingface.co/torchao-testing/opt-125m-Float8DynamicActivationFloat8WeightConfig-v1-0.13.dev
-    (
-        "torchao-testing/opt-125m-Float8DynamicActivationFloat8WeightConfig-v1-0.13.dev",
-        1,
-        "Float8DynamicActivationFloat8WeightConfig",
-    ),
+    # skipping for now, not sure why it fails, also we are removing this
```

Comment:
Note: these fails when we check for exact match of results, could be something related to transformer versions